js/App.js

```diff
@@ -125,11 +126,43 @@ App.prototype.setUnreadChatMessageCount = function(count) {
   }
 };
 
-App.prototype.intlNumFormat = function(numberToFormat, maxDigits){
-  maxDigits = maxDigits || 8; //default to show down to the satoshi (.00000001)
+App.prototype.intlNumFormat = function(numberToFormat, maxDigits = 8) {
   return new Intl.NumberFormat(window.lang, {maximumFractionDigits: maxDigits}).format(numberToFormat);
 };
 
+App.prototype.getBitcoinUnit = function() {
+  if (!this._bitcoinUnit) {
+    this._bitcoinUnit = localStorage.getItem('BitcoinUnit') || 'BTC';
+  }
+  return this._bitcoinUnit;
+};
+
+App.prototype.setBitcoinUnit = function(unit) {
+  this._bitcoinUnit = unit;
+  localStorage.setItem('BitcoinUnit', unit);
+};
+
+/**
+ * Format a bitcoin amount in the user's locale.
+ *
+ * @param {Number} amount - Bitcoin (BTC) amount.
+ * @param {Number} [maxDigits=8] - Maximum number of fraction digits to display.
+ * @returns {String} localised amount ending with ' BTC' or the default bitcoin unit.
+ *
+ * @see intlNumFormat
+ */
+App.prototype.formatBitcoin = function(amount, maxDigits) {
```

Comment:
For the NaN bug, I think it's happening because in several places the data can be undefined. Those are probably the spots in the template that have an `|| 0` as a backup if the number formatting fails. 

The easiest way to test transactions is to set up several testnet nodes. 

1, set your server to use testnet. 
`python openbazaard.py start -t`

2. set up a second client and server. You can do this on a VM, with a cloud server, on a 2nd computer, etc.

3. create a moderated transaction between server A and server B. You can use my tesnet node as a moderator (bce5e1035bc47d5400429d1fa2f1e0d05ffff37e). It normally runs 24/7.

4. You will need some testnet coins. The easiest way to get testnet coins is to download Copay (www.copay.io), create a new testnet wallet (when creating a wallet look in the advanced settings), and get some testnet coins from testnet faucets (google "testnet faucets, there are several out there).

This will let you rapidly test transactions. It's important to check the entire transaction flow, including disputes, to make sure everything still works. 

In v2 there are more automated tests, for v1 it's still a fairly manual process. 

Thanks for your work so far, we appreciate it.